packages/ui-library/readme.md

```diff
@@ -1,7 +1,7 @@
 # Welcome to the Yoast UI Library
 A React component library for building Yoast user interfaces. Please visit [the Storybook](https://ui-library.yoast.com/) for an interactive overview of all available components and examples on how to use them.
 
-[View the 3.0 changelog here](https://github.com/Yoast/wordpress-seo/blob/feature/ui-library-3/packages/ui-library/changelog.md).
+[View the 3.0 changelog here](https://github.com/Yoast/wordpress-seo/blob/trunk/packages/ui-library/changelog.md).
```

Comment:
Maybe we should not even mention the version here? Saves some updating 😉 

```suggestion
[View the changelog here](https://github.com/Yoast/wordpress-seo/blob/trunk/packages/ui-library/changelog.md).
```

Reply:
Fair point! I also tried to link it to the `changelog` page, but that proved more difficult than I thought. So left it at this.